### DISCUSSION TITLE
CRXJS and Turborepo

### DISCUSSION THREAD

**Discussion by nicolasthy** [crxjs/chrome-extension-tools#840] [Q&A]
Hello 👋 
Has anyone tried to implement CRXJS with Turborepo?
I have a working chrome extension, but when I apply changes to a shared component in the monorepo it doesn't apply to the extension unless if I restart the server. Am I missing something in the config?

I do see the following in the console, but nothing changes in the browser:
```
extension:dev: 7:19:58 PM [vite] hmr update /@fs/Users/nicolasthy/Sites/playground/x-turborepo/packages/ui/card.tsx
```

**Comment by salmin89** [comment]
did you ever figure this out?
